docs/cookbook/fromLegacyDate.mjs

```diff
@@ -0,0 +1,31 @@
+const legacyDate = new Date('1970-01-01T00:00:01Z');
+const instant = legacyDate.toTemporalInstant();
+
+assert.equal(instant.toString(), '1970-01-01T00:00:01Z');
+
+// If you need a ZonedDateTime, use the toZonedDateTime() or
+// toZonedDateTimeISO() method of the resulting Instant.
+
+// You will need to specify a time zone. You might not have time
+// zone information, since legacy Date is an exact time without
```

Comment:
I'd remove the rest of this comment.  (Too bad GH doesn't allow multi-line suggestions!)

Reply:
> (Too bad GH doesn't allow multi-line suggestions!)

It does, it's just not entirely obvious that it's possible.

1. Click and drag on the line numbers to select the lines you want.
    1. Alternatively, click on the line number for the first line of the block, then shift+click on the line number for the last line of the block.
2. Click on the normal "+" icon for leaving a comment, but only on the last line of the block.
3. Making a suggestion should now cover all the selected lines.